WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -17,7 +17,7 @@ final class CompliancePopoverCoordinator: CompliancePopoverCoordinatorProtocol {
     }
 
     func presentIfNeeded() {
-        guard FeatureFlag.compliancePopover.enabled else {
+        guard FeatureFlag.compliancePopover.enabled, !defaults.didShowCompliancePopup else {
```

Comment:
_This is unrelated to #21058_ 
This change avoids frequent unnecessary network calls

Reply:
Ah, good catch